.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+    push:
+      branches: master
+    pull_request:
+      branches: master
+jobs:
+# This workflow contains a single job called "build"
+  ci:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    # To save time, detach and pull some of the largest docker files
+    - run: (docker pull mongo:4.2.3-bionic && docker pull mongo:4.2.3-bionic) &
```

Comment:
Is this pulling `mongo:4.2.3-bionic` twice?

Reply:
Oh, that's a typo, it should be the Cypress container in addition to Mongo.